spec/requests/device_requests_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'Request devices', :type => :request do
+  describe 'PUT #user_update' do
+    before(:each) { @device = FactoryGirl.create(:device) }
+    before(:each) { @log = FactoryGirl.create(:JsonLogDevice, device_id: @device.id) }
+
+    it 'sends request succesfully' do
+      put "/devices/#{@device.id}/stat", device: FactoryGirl.attributes_for(:device)
+      expect(response).to have_http_status(200)
+    end
+
+    it 'finds correct device type and data' do
+      put "/devices/#{@device.id}/stat", device: FactoryGirl.attributes_for(:device, type: 'Fridge')
+      assigns(:device)['type'].should eq(@device.type)
```

Comment:
not sure that this test is working
it will be better: `expect(@device.reload.type).to eq('Fridge')